### ISSUE TITLE
Minor issues found during testing this out

### ISSUE THREAD

**Issue by axlroden** [NatiSayada/k3s-proxmox-terraform-ansible#14]
You spelled template wrong many places, example: 
https://github.com/NatiSayada/k3s-proxmox-terraform-ansible/blob/8763038eb88233ab1208484512eaa5c95febae0f/terraform/variables.tfvars.sample#L10

There are no gateway/netmask settings in the example tfvars file, resulting in wrong "default" gateway if you change ip's to a different subnet.
Just a small annoyance if you dont spot it the first time, but could be a major time sink for some.

**Comment by clibequilibrium** [comment]
@axlroden 

> 
> There are no gateway/netmask settings in the example tfvars file, resulting in wrong "default" gateway if you change ip's to a different subnet.
> Just a small annoyance if you dont spot it the first time, but could be a major time sink for some.

Could you elaborate on that and what would be the correct steps ?

**Comment by NatiSayada** [comment]
@axlroden makes sense that if you can change the IP addresses, you will also be able to update the gateway IP.
See the #15 PR and let me know if that will work for you

**Comment by axlroden** [comment]
sure, but your not putting in netmask either, if someone is not doing /24

**Comment by axlroden** [comment]
everything else looks good, same way I handled it.

**Comment by NatiSayada** [comment]
@axlroden tfvars has been updated with the netmask and I reorder it to make more sense.
